### DISCUSSION TITLE
UI 界面设计太丑陋，建议优化

### DISCUSSION THREAD

**Discussion by melin** [apache/gravitino#5274] [Ideas]
UI 界面设计太丑陋，字体大小、UI组件高度看起来不是舒服，flink web ui 就比较好看，建议参考一下。
gravitino 更多操作依赖webui，好ui设计，也有利于gravitino推广，

**Comment by melin** [comment]
cc @jerryshao 有这方面优化计划。

**Reply by shaofengshi** [reply]
Hello @melin , we're very lack of the development resources on Web GUI. Community contribution is welcomed. If you can contribute, that would be good!
